airflow/settings.py

```diff
@@ -18,40 +18,11 @@
 from __future__ import unicode_literals
 
 import logging
-import os
 import sys
 
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
 
-from airflow import configuration as conf
-
-
-class DummyStatsLogger(object):
-    @classmethod
-    def incr(cls, stat, count=1, rate=1):
-        pass
-    @classmethod
-    def decr(cls, stat, count=1, rate=1):
-        pass
-    @classmethod
-    def gauge(cls, stat, value, rate=1, delta=False):
-        pass
-
-Stats = DummyStatsLogger
-
-if conf.getboolean('scheduler', 'statsd_on'):
-    from statsd import StatsClient
-    statsd = StatsClient(
-        host=conf.get('scheduler', 'statsd_host'),
-        port=conf.getint('scheduler', 'statsd_port'),
-        prefix=conf.get('scheduler', 'statsd_prefix'))
-    Stats = statsd
-else:
-    Stats = DummyStatsLogger
```

Comment:
Move DummyStatsLogger from settings.py to configuration.py to avoid import loop between both files.